example/package.json

```diff
@@ -1,46 +1,37 @@
 {
   "name": "dapp-cookie-cutter",
-  "version": "0.1.0",
   "private": false,
+  "version": "0.1.0",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/secretkeylabs/dapp-cookie-cutter"
   },
-  "license": "ISC",
   "description": "A cookie cutter for dapps using sats-connect",
   "scripts": {
-    "prepare": "cd .. && npm i && npm run build && cd example",
-    "start": "GENERATE_SOURCEMAP=false PORT=3001 react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject"
+    "dev": "vite",
```

Comment:
since we are updating the scripts we should update the command to run the example app from the root folder https://github.com/secretkeylabs/sats-connect/pull/145#pullrequestreview-2166031673